.changeset/thick-chicken-swim.md

```diff
@@ -0,0 +1,5 @@
+---
+'@directus/api': patch
+---
+
+Added falsy fallback condition to search query
```

Comment:
```suggestion
```md
Fixed the API to not return any items for search queries that do not match any field types

::: notice
Previously, the API returned all items for collections that had no applicable fields to a search query. This has now been corrected so that the API does not return any items in such a case.
:::
```